usaspending_api/api_docs/api_documentation/download/download_status.md

```diff
@@ -0,0 +1,40 @@
+## [Download Status](#usaspending-api-documentation)
+**Route:** `/api/v2/download/status/`
+
+**Method:** `GET`
+
+This route gets the current status of a download job that _that has been requested_ with the `v2/download/awards/` or `v2/download/transaction/` endpoint _that same day_.
```

Comment:
Sorry, italics in my comment were just to show the changes. Italicized text can be un-styled.

Reply:
Done